src/main/java/edu/usf/cutr/tba/manager/TravelBehaviorDataAnalysisManager.java

```diff
@@ -239,10 +260,99 @@ private void completeTravelBehaviorRecord(TravelBehaviorInfo tbi, QueryDocumentS
             mLastTravelBehaviorRecord.setOriginDestinationDistance(distance);
         }
 
-        String regionId = findRegionIdFromDeviceInfoList(doc, userDeviceInfoList);
-        mLastTravelBehaviorRecord.setRegionId(regionId);
+        //String regionId = findRegionIdFromDeviceInfoList(doc, userDeviceInfoList);
+        //mLastTravelBehaviorRecord.setRegionId(regionId);
+
+        //While completing TBR update isIgnoringBatteryOptimization, isTalkBackEnabled
+        //and isPowerSaveModeOn
+        if(mLastTravelBehaviorRecord.getActivityEndTimeMillis() != null){ /*if valid activity end time*/
```

Comment:
Let's use the normal IntelliJ template for formatting new code (you can format within IntelliJ), which adds spaces as follows:
* `if (...) {`
* `// comment`

Also, I'd strongly suggest you check the "Optimize imports" checkbox when committing within IntelliJ to ensure the import statements are always processed.